src/vue/common/TableCustom.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
Why do we need the wrap the table in the component? It looks prettier for me to use the default html table with class `app__table` defined in app.scss

Reply:
Resolved, updated